scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -47,7 +44,7 @@ workflow GvsCalculatePrecisionAndSensitivity {
     }
   }
 
-  String output_chr_basename = if defined(contig) then output_basename + "." + contig else output_basename
+  String output_chr_basename = if defined(contig) then output_basename + "." + select_first([contig]) else output_basename
```

Comment:
what does this do? You are making it into an array and getting the first item?
select_first([contig])

 Don't you already know it's defined based on the if statement?

Reply:
It failed validation with miniwdl (trying to append a String and a String?. It's possible it could be None/undefined, so the select_first([contig]) tries to make it a String. It's passed a (here one element) array of elements and tries to find the first defined element in the array. If it fails, it will be a run time error, which won't ever happen because of the if defined, but I do this trick to silence the miniwdl check.